src/common/prntbase.cpp

```diff
@@ -1944,6 +1955,11 @@ void wxPrintPreviewBase::CalcRects(wxPreviewCanvas *canvas, wxRect& pageRect, wx
     float screenPrintableWidth = zoomScale * m_pageWidth * m_previewScaleX;
     float screenPrintableHeight = zoomScale * m_pageHeight * m_previewScaleY;
 
+#ifndef wxHAS_DPI_INDEPENDENT_PIXELS
```

Comment:
Sorry, it bothers me that we still need to check for `wxHAS_DPI_INDEPENDENT_PIXELS` explicitly, this is not supposed to be necessary with the various DIP-related helpers.

AFAICS you do this here because `FromDIP()` doesn't work with float values, but couldn't we just use `FromDIP(m_pageWidth)` and `FromDIP(m_pageHeight)` when computing `screenPrintable{Width,Height}` above instead?

Reply:
Changed to use `FromDIP()` as suggested.